sdk/loadtestservice/azure-analytics-loadtestservice/swagger/README.md

```diff
@@ -0,0 +1,16 @@
+### Settings
+
+```yaml
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/loadtestservice.json
+output-folder: ../azure/analytics/loadtestservice
+namespace: azure.analytics.loadtestservice
+package-name: azure-analytics-loadtestservice
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: true
+no-namespace-folders: true
+title: LoadTestClient
+version-tolerant: true
```

Comment:
nit: no need for `version-tolerant: true` anymore bc we now default to version tolerant

Reply:
Resolved